core/camel/src/main/java/org/opennms/core/camel/MinionDTO.java

```diff
@@ -0,0 +1,118 @@
+/*******************************************************************************
```

Comment:
There's already DTO for Minion , may be you can use that, https://github.com/OpenNMS/opennms/blob/develop/features/minion/heartbeat/common/src/main/java/org/opennms/minion/heartbeat/common/MinionIdentityDTO.java